rice-middleware/impl/src/main/java/org/kuali/rice/kew/impl/document/search/DocumentSearchCriteriaBoLookupableHelperService.java

```diff
@@ -591,10 +591,11 @@ public HtmlData getInquiryUrl(BusinessObject bo, String propertyName) {
         if (StringUtils.isBlank(principalId) ) {
             return link;
         }
-        link.setTarget(this.targets.getRouteLogTarget(documentType.getName()));
         link.setDisplayText("Initiator Inquiry for User with ID:" + principalId);
-        String url = ConfigContext.getCurrentContextConfig().getProperty(Config.KIM_URL) + "/" +
-                "identityManagementPersonInquiry.do?principalId=" + principalId;
+        String url = ConfigContext.getCurrentContextConfig().getProperty(Config.KFS_URL) + "/" +
+                "kr/inquiry.do?businessObjectClassName=org.kuali.kfs.kim.impl.identity.PersonImpl&principalId="
+                + principalId
+                + "&methodToCall=start";
```

Comment:
do we want these inquiry URLs to be modal? I know we had to tweak the doc header inquiry link for that, see: https://github.com/KualiCo/financials/blob/78e082627281a0b31c72e51c47bd7427fe99a3bb/kfs-core/src/main/java/org/kuali/kfs/kns/web/struts/form/KualiDocumentFormBase.java#L256-L276

Reply:
when i tried to do it in a modal, it looked really odd as the styling did not get applied, and then it is just a modal over the portal.

Reply:
yeah there were some hoops we had to jump through for that